languages/es.json

```
@@ -160,6 +163,9 @@
   "building-workshops-info3C": null,
   "host-header": "Organizando un evento NodeSchool",
   "host-info": "NodeSchool es un esfuerzo impulsado completamente por la comunidad y es 100% open source. Esto significa que cualquiera puede utilizar los materiales de NodeSchool y/o organizar un evento.",
+  "host-info-cost": "La única expectativa para los organizadores es que los eventos sean sin fines de lucro(gratuitos o de bajo costo) y abiertos para la administración de eventos (para que nadie pueda \"controlar\" los eventos).",
+  "host-info-wiki": "Creamos el <a href=\"https://github.com/nodeschool/organizers/wiki\">Wiki de Organizadores</a> para que el organizador conserve el conocimiento de los capítulos organizados en todo el mundo.",
```

Let's use "la wiki" to be consistent with the sentence below, like:
```suggestion
  "host-info-wiki": "Creamos la <a href=\"https://github.com/nodeschool/organizers/wiki\">Wiki de Organizadores</a> para que el organizador tenga conocimiento de los capítulos organizados alrededor del mundo.",
```